core/src/test/resources/conf/application-test.properties

```diff
@@ -0,0 +1,6 @@
+db.driver=org.h2.Driver
+db.url=jdbc:h2:~/phoneshop/db/h2
```

Comment:
Regarding in-memory configuration, please see https://stackoverflow.com/a/51824325

Reply:
Fixed, thanks a lot!